samples/multiplatform/src/jvmTest/spek/src/test/kotlin/SmokeTest.kt

```diff
@@ -0,0 +1,31 @@
+package org.spekframework.spek2
+
+import ch.tutteli.atrium.api.fluent.en_GB.messageContains
+import ch.tutteli.atrium.api.fluent.en_GB.toBe
+import ch.tutteli.atrium.api.fluent.en_GB.toThrow
+import ch.tutteli.atrium.api.verbs.expect
+import org.spekframework.spek2.style.specification.describe
+import java.lang.IllegalArgumentException
+
+object CalculatorTestBySpek : Spek({
+    describe("Calculator") {
+        val calculator by memoized { Calculator() }
+
+        it("1 + 2 = 3") {
+            val sum = 3
```

Comment:
change to 
```suggestion
            val sum = 2
```
so that you see the test fails in the build